src/main/java/eggventory/model/PersonList.java

```diff
@@ -89,4 +92,46 @@ public static int findPerson(String matricNo) {
         return -1;
     }
 
+    //@@author Deculsion
+    public static int getSize() {
+        return personList.size();
+    }
+
+    /**
+     * Creates a tablestruct of all persons in personlist.
+     * @return A tablestruct object.
+     */
+    public static TableStruct getAllPersonStruct() {
+        TableStruct tableStruct = new TableStruct("Person List");
+        tableStruct.setTableColumns("Matriculation No.", "Full Name");
+
+        ArrayList<ArrayList<String>> dataArray = new ArrayList<>();
+        for (Person person: personList) {
+            dataArray.add(person.getDataAsArray());
+        }
+
+        tableStruct.setTableData(dataArray);
+
+        return tableStruct;
+    }
+
+    /**
+     * Converts the person list to a string.
+     * @return A string representing all data in personlist.
+     */
```

Comment:
Not sure, but does this also have the similar concern regarding violation of SLAP like in the example discussed for String output = "" above.